Examples/Scripts/Python/geant4_parallel.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+from pathlib import Path
+from multiprocessing import Pool
+from functools import partial
+
+import numpy as np
```

Comment:
We don't typically rely on thirdparty libs in the scripts directory. I suppose this is doable without numpy.

Reply:
I guess we can get rid of numpy here because we only use it to hand out the event numbers. not sure how critical this is

Reply:
Yeah it's just kind of handy that numpy provides a function that does exactely what I want here, but of course that can be done also quite easily by hand